docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,442 @@
+|  submitter_sequencing\_ experiment_id  | experiment |  ![Required](/assets/submission/dictionary-required.svg)  |  The unique identifier of the sequencing experiment.                                                     |                                    |  |
+|  platform                              | experiment |  ![Required](/assets/submission/dictionary-required.svg)  |  The sequencing platform type used in data generation.                                                   | CAPILLARY, LS454, ILLUMINA, SOLID, HELICOS, IONTORRENT, ONT, PACBIO, Nanopore, BGI |  |
+|  platform_model                        | experiment |                                                           |  The model number of the sequencing machine used in data generation.                                     |                                    |  |
+|  experimental_strategy                 | experiment |  ![Required](/assets/submission/dictionary-required.svg)  |  The primary experimental method. For sequencing data it refers to how the sequencing library was made.  |  WGS, WXS, RNA-Seq, Bisulfite-Seq  |  |
+|  sequencing_date                       | experiment |                                                           |  Date sequencing was performed.      |  |  |
+|  read_group_count  |  |  ![Required](/assets/submission/dictionary-required.svg)  |  The number of read groups in the molecular files being submitted submitted.                                                                                                               |   |  |
+| read_group_id_in_bam | read_groups |   |  Optional field indicating the read group id that is in the BAM.  This cannot be submitted for FASTQ files. |                                                                      |  |
+|  submitter_read\_ group_id  | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  The unique identifier of a read group; must be unique within each payload.                                                                                                                                                                                                                                                                                         |                      |  |
+|  platform_unit              | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  Unique identifier including the {FLOWCELL_BARCODE}.{LANE}.{SAMPLE_BARCODE}. The {FLOWCELL_BARCODE} refers to the unique identifier for a particular flow cell. The {LANE} indicates the lane of the flow cell and the {SAMPLE_BARCODE} is a sample/library-specific identifier. For non-multiplex sequencing, platform unit and read group have a one-to-one relationship.  |                      |  |
```

Comment:
should you give examples for complicated ones like this?

Reply:
er... no its very data/platform dependent.